email/finalizer.go

```diff
@@ -230,3 +224,62 @@ func (f *Finalizer) threadedFinalizeMessages() {
 		}
 	}
 }
+
+// sendAbuseReport sends the abuse report for the given abuse email to the given
+// email address. This is extracted in a standalone function for unit testing
+// purposes.
+func sendAbuseReport(client *client.Client, email database.AbuseEmail, mailbox, to string) error {
+	// generate a uuid as message id
+	var u *uuid.UUID
+	u, err := uuid.NewV4()
+	if err != nil {
+		return errors.AddContext(err, "failed to generate uid")
+	}
+
+	// construct the email message
+	msg := fmt.Sprintf("Subject: Re: %s\n", email.Subject)
+	msg += fmt.Sprintf("Message-ID: <%s@abusescanner>\n", u)
+	msg += fmt.Sprintf("References: %s\n", email.MessageID)
+	msg += fmt.Sprintf("In-Reply-To: %s\n", email.MessageID)
+	msg += fmt.Sprintf("From: SCANNED <%s>\n", scannerEmailAddress)
+	msg += fmt.Sprintf("To: %s\n", to)
+	msg += ""
```

Comment:
Is this supposed to be `"\n"`? Because otherwise this isn't really doing anything right?

Reply:
lol